spec/bolt/inventory_spec.rb

```diff
@@ -61,120 +61,6 @@ def targets(names)
     }
   }
 
-  describe :config_for do
```

Comment:
these specific search merge behavior tests need to go somewhere unless they were already dupes. Which is possible since I got more systematic about writing them at the end and may not have cleaned everything up

Reply:
I suggested rewriting them as tests using `get_targets`, that verifies the final `Target` objects. We need that anyway because our tests don't match the `Target` contract very well right now.